docs/reference/search/search-your-data/search-with-synonyms.asciidoc

```diff
@@ -75,6 +75,13 @@ A large number of inline synonyms increases cluster size unnecessarily and can l
 
 Once your synonyms sets are created, you can start configuring your token filters and analyzers to use them.
 
+[WARNING]
+======
+Synonym sets must exist before they can be added to indices.
```

Comment:
Shouldn't it be "synonyms sets" to be consistent with the rest of the documentation?

Reply:
++